### PR TITLE
Update Strapi Quickstart 

### PR DESCRIPTION
**Pull request by davidkartuzinski**
#### Description of what you did:

Update the Strapi Quickstart:

1. Change to yarn/npx commands (from Strapi Global Installation)
2. Remove unnecessary steps ( no longer part of the process) 
3. Clarified some steps
4. Removed Article link in the bottom (no longer exists)
5. Other minor changes

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [x] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [ ] Admin
- [x] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [x] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

File: docs/3.0.0-beta.x/getting-started/quick-start.md [hunk 1]
```diff
@@ -1,118 +1,117 @@
...
 
-## 1. Install Strapi globally
+(Before continuing, please make sure [Node.js and npm are properly installed](install-requirements.html) on your machine.)
 
-Please make sure [Node.js and npm are properly installed](install-requirements.html) on your machine.
+## 1. Install Strapi and Create a new project
 
 ```bash
-npm install strapi@beta -g
+yarn create strapi-app my-project --quickstart
```
Comment (alexandrebodin) [inline docs/3.0.0-beta.x/getting-started/quick-start.md]: You should link the yarn installation doc like your are doing with node and npm

File: docs/3.0.0-beta.x/getting-started/quick-start.md [hunk 2]
```diff
@@ -1,118 +1,117 @@
...
+  - Click the **String** field
+  - Type `name` in the **Name** field
+  - Click over to the **ADVANCED SETTINGS** tab, and check the `Required field` and the `Unique field`
+  - Click the **"+ Add New Field"** button
+  - Click the **Rich Text** field
+  - Type `description` under the **BASE SETTINGS** tab, in the **Name** field
+  - Click `Done`
+- Click the **Save** button and wait for Strapi to restart
+
+## 4. Create another new Content type
```
Comment (alexandrebodin) [inline docs/3.0.0-beta.x/getting-started/quick-start.md]: Create another new seems redundant ?

File: docs/3.0.0-beta.x/getting-started/quick-start.md [hunk 3]
```diff
@@ -1,118 +1,117 @@
...
-Navigate to [**CONTENT TYPES** - **Restaurants**](http://localhost:1337/admin/plugins/content-manager/restaurant?source=content-manager).
-
-- Click on **+ Add New Restaurant** button. Type `Strapi Restaurant` in the **Restaurant** field. Type `Strapi restaurant is a cosy restaurant delivering one of the very fastest and nicest dining experiences in the world, combining nods to tradition with fierce modernity, warmth with daring.` into the **Description** field.
+  - Click the **String** field
+  - Type `name` under the **BASE SETTINGS** tab, in the **Name** field
+  - Click over to the **ADVANCED SETTINGS** tab, and check the `Required field` and the `Unique field`
+  - Click the **"+ Add New Field"** button
+  - Click the **Relation** field
+  - On the right side, click the **Permissions** dropdown and select, `Restaurant`
+  - In the center, select the icon the represents `many-to-many`. The text should read, `Categories has and belongs to many Restaurants`
```
Comment (alexandrebodin) [inline docs/3.0.0-beta.x/getting-started/quick-start.md]: the icon `that`

File: docs/3.0.0-beta.x/getting-started/quick-start.md (hunk 1 above)
Comment (davidkartuzinski) [reply docs/3.0.0-beta.x/getting-started/quick-start.md]: Done.

File: docs/3.0.0-beta.x/getting-started/quick-start.md (hunk 2 above)
Comment (davidkartuzinski) [reply docs/3.0.0-beta.x/getting-started/quick-start.md]: I agree. After rereading it wasn't very clear.  I made both steps #3 and #4 more clear. Good catch.

File: docs/3.0.0-beta.x/getting-started/quick-start.md (hunk 3 above)
Comment (davidkartuzinski) [reply docs/3.0.0-beta.x/getting-started/quick-start.md]: Thank you.

**Comment by davidkartuzinski** [comment]
> Just a few comments

@alexandrebodin I have resolved your comments. Thank you.

**Pull request merged**